src/SFA.DAS.EmployerDemand.Application/Demand/Queries/GetProviderEmployerDemand/GetProviderEmployerDemandQueryHandler.cs

```diff
@@ -28,7 +33,9 @@ public async Task<GetProviderEmployerDemandQueryResult> Handle(GetProviderEmploy
                 SelectedCourseId = request.CourseId,
                 SelectedLocation = result.Location,
                 SelectedRadius = request.LocationRadius,
-                Ukprn = request.Ukprn
+                Ukprn = request.Ukprn,
+                Sectors = result.Sectors.Select(c => c.Route).ToList(),
```

Comment:
tbh this makes me cry a bit on the inside as well since sectors and routes are mentioned on the same line of code.